ktor-client/ktor-client-okhttp/jvm/src/io/ktor/client/engine/okhttp/OkHttpEngine.kt

```diff
@@ -144,7 +144,9 @@ public class OkHttpEngine(override val config: OkHttpConfig) : HttpClientEngineB
     private fun createOkHttpClient(timeoutExtension: HttpTimeoutConfig?): OkHttpClient {
         val builder = (config.preconfigured ?: okHttpClientPrototype).newBuilder()
 
-        builder.dispatcher(Dispatcher())
+        if (config.preconfigured == null) {
+            builder.dispatcher(Dispatcher())
```

Comment:
Probably we should just drop this line as `OkHttp` already [creates `Dispatcher` by default](https://github.com/square/okhttp/blob/parent-5.3.2/okhttp/src/commonJvmAndroid/kotlin/okhttp3/OkHttpClient.kt#L587):
```kotlin
internal var dispatcher: Dispatcher = Dispatcher()
```